File: Source/GraphAlgorithms/dfs.js
function dfs(node, visited, G) {
    visited[node] = true;
    console.log(node);

    for(let i = 0; i < G[node].length; i++) {
        if(!visited[G[node][i]]) {
            dfs(G[node][i], visited, G);
        }
    }
}

File: Source/GraphAlgorithms/dfs.c
void dfs(int node, int visited[], int G[][N]) {
    visited[node] = 1;
    printf("%d ", node);

    for(int i = 0; i < N; i++) {
        if(G[node][i] == 1 && visited[i] == 0) {
            dfs(i, visited, G);
        }
    }
}
